internal/workspaces/notebooks/notebooks.go

```diff
@@ -0,0 +1,161 @@
+package notebooks
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	ihttp "github.com/llm-operator/cli/internal/http"
+	"github.com/llm-operator/cli/internal/runtime"
+	"github.com/llm-operator/cli/internal/ui"
+	jv1 "github.com/llm-operator/job-manager/api/v1"
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+)
+
+const (
+	path = "/workspaces/notebooks"
+)
+
+// Cmd is the root command for notebooks.
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                "notebooks",
+		Short:              "Notebook commands",
+		Aliases:            []string{"nbs", "nb"},
+		Args:               cobra.NoArgs,
+		DisableFlagParsing: true,
+	}
+	cmd.AddCommand(createCmd())
+	cmd.AddCommand(listCmd())
+	cmd.AddCommand(getCmd())
+	return cmd
+}
+
+func createCmd() *cobra.Command {
+	var (
+		name    string
+		imgType string
+	)
+	cmd := &cobra.Command{
+		Use:  "create",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return create(cmd.Context(), name, imgType)
+		},
+	}
+	cmd.Flags().StringVar(&name, "name", "", "Name of the API key")
+	cmd.Flags().StringVar(&imgType, "image-type", "jupyter-lab-base", "Type of the Notebook Image")
+	_ = cmd.MarkFlagRequired("name")
+	return cmd
+}
+
+func listCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:  "list",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return list(cmd.Context())
+		},
+	}
+}
+
+func getCmd() *cobra.Command {
+	var (
+		id string
+	)
+	cmd := &cobra.Command{
+		Use:  "get",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return get(cmd.Context(), id)
+		},
+	}
+	cmd.Flags().StringVar(&id, "id", "", "ID of the notebook")
```

Comment:
No need to change now, but we probably should support lookup with `name`. The backend also probably have `name` as DB column and have a unique constraint on `(name, projectID)`?

Reply:
Btw, fine-tuning jobs don't have names as it is not in the OpenAI API spec, but I'm wondering if we should have that somehow.

Reply:
I feel thinking about the name every time is a bit annoying as a job tends to have a short lifetime. That said an optional name or description would be useful.

Reply:
Agreed!